training/distributed_training/pytorch/model_parallel_v2/shared-scripts/train_utils.py

```diff
@@ -43,7 +43,7 @@ def compute_tflops(throughput, num_params, world_size, seq_len):
     Based on the formula in
     https://developer.nvidia.com/blog/scaling-language-model-training-to-a-trillion-parameters-using-megatron/
     """
-    return 6 * throughput * num_params / world_size * seq_len * 1e-12
+    return 6 * throughput * num_params / dp_size * seq_len * 1e-12
```

Comment:
Should we change this tflops calculation to be the one that supports MoE?

Reply:
Updated